.storybook/stories/label/label.stories.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Parameters } from '@storybook/addons';
+import { Story } from '@storybook/angular';
+
+import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+
+enum LabelType {
+  Default = '',
+  Purple = 'label-purple',
+  Blue = 'label-blue',
+  Orange = 'label-orange',
+  LightBlue = 'label-light-blue',
+  Info = 'label-info',
+  Success = 'label-success',
+  Warning = 'label-warning',
+  Error = 'label-danger',
+}
+
+const defaultStory: Story = args => ({
+  template: `
+    <span class="label" [class.clickable]="clickable" [ngClass]="labelType">
+      {{content}}
+      <span *ngIf="badgeContent" class="badge">{{badgeContent}}</span>
+    </span>
+  `,
+  props: { ...args },
+});
+
+const defaultParameters: Parameters = {
+  title: 'Label/Label',
```

Comment:
Since labels are closely related to forms, should this be `Forms/Label`?

Reply:
They're clickable buttons that are styled differently. Not related to forms